statistics-production/api-data-standards.qmd

```diff
@@ -0,0 +1,191 @@
+|------------------------|------------------------|------------------------|--------------------------|--------------------------|-------------------------|----------------------|------------------------|-----------------|----------------------|------------------------|-----------------|
+| 30                     | 40                     | 50                     | 60                       | 80                       | 100                      | 0.2                  | 0.21                   | 0.21            |0.09                  | 0.08                   | 0.10            |
+
+: Attainment grade rates and scores in non-tidy format
+
+:::
+
+#### Example of good practice
+
+The following would be accepted for publication via the API. In this case, splitting the data into separate data files is required in order to create tidy data structures.
+
+::: {.table-responsive}
+
+|attainment_metric   | count_pupils | percent_pupils |
+|--------------------|--------------|----------------|
+| Grades 9-5         | 30           | 60             |          
+| Grades 9-4         | 40           | 80             |         
+| Grades 9-1         | 50           | 100            |                  
+
+: Attainment grade rates in tidy format
+
+:::
+
+::: {.table-responsive}
+
+| sex    | attainment_metric | score_average  |
+|--------|-------------------|----------------|
+| Female | Progress 8        | 0.21           |          
+| Male   | Progress 8        | 0.20           |         
+| Total  | Progress 8        | 0.21           |                  
+| Female | Attainment 8      | 0.08           |          
+| Male   | Attainment 8      | 0.09           |         
+| Total  | Attainment 8      | 0.08           |                  
+
+: Attainment scores in tidy format
+
+:::
+
+### Example 2 - Pupil counts, percents and characteristics
+
+#### Example of bad practice
+
+::: {.table-responsive}
+
+| count_schools | count_pupils_male | count_pupils_female | count_pupils_total | percent_pupils_male | percent_pupils_female | percent_pupils_total |
+|---------------|-------------------|---------------------|--------------------|---------------------|-----------------------|----------------------|
+| 2             | 120               | 130                 | 250               | 48                  | 52                    | 100                  |
+
+: Pupil counts and percentages in non-tidy format
+
+:::
+
+#### Example of good practice
+
+::: {.table-responsive}
+
+| sex                | count_schools | count_pupils | percent_pupils |
+|--------------------|---------------|--------------|----------------|
+| Male               | 2             | 30           | 60             |          
+| Female             | 2             | 40           | 80             |         
+| Total              | 2             | 50           | 100            |                  
+
+: Pupil counts and percentages in tidy format
+
+:::
+
+###
```

Comment:
Blank heading?

Reply:
Must have accidentally deleted it. It used to be summary I think and have added it back in.